### DISCUSSION TITLE
Sometimes I am unable to manually sideload an add-in to Office on the web

### DISCUSSION THREAD

**Discussion by prerit714** [OfficeDev/office-js#4006] [Q&A] [answered]
![more-add-in](https://github.com/OfficeDev/office-js/assets/62067658/167f7ff8-f003-4a28-9d12-1ce21f5d2c5b)
Sometimes after clicking on *More Add-ins* button, the modal which opens (which allows us to upload a manifest file) next does not open at all. What could be the possible reason for this? Any help is appreciated. Thanks.

**Comment by zhenhuangMSFT** [comment]
Hi @prerit714 , could you please share us your session id? It could be found under the `File` tab, in `About` section.

**Reply by prerit714** [reply, marked as answer]
I have been trying to recreate this issue. It works perfectly fine as for now. But if it ever happens again I will share the details here. Thanks for the reply.
